hudi-client/hudi-client-common/src/main/java/org/apache/hudi/config/HoodieCompactionConfig.java

```diff
@@ -234,6 +235,12 @@ public class HoodieCompactionConfig extends HoodieConfig {
           + "the record payload class to merge records in the log against each other, merge again with the base file and "
           + "produce the final record to be written after compaction.");
 
+  public static final ConfigProperty<String> COMBINE_ENGINE_CLASS_NAME = ConfigProperty
+      .key("hoodie.compaction.combine.engine.class")
+      .defaultValue(HoodieAvroRecordCombiningEngine.class.getName())
+      .withDocumentation("Combine engine class to use for performing compactions, i.e merge delta logs with current base file and then "
```

Comment:
This needs to be made clearer IMO. Can we make the doc more about what a "combine engine" is, rather than explaining what compaction does

Reply:
Yeah,I will make adjustments, Thank you for your advice.